src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -226,11 +227,9 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
           schema = @Schema(implementation = QueryObject.class))
           QueryObject queryObject) {
     try {
-      if (auth == null || !checkBasicAuth(auth, "icinga.username", "icinga.password")) {
-        return Response.status(Response.Status.UNAUTHORIZED).build();
-      }
       return Response.ok(
-          SearchController.releaseQuery(new Gson().toJson(queryObject.getCqlQueryList()),
+          SearchController.releaseQuery(new Gson().toJson(queryObject.getCqlQueryList().getQueries()
+                  .toArray(new CqlQuery[0])),
```

Comment:
Why do we need an array here?

Reply:
because i changed in the storeCriteria method in the class InquiryHandler.
see src/main/java/de/samply/share/broker/rest/InquiryHandler.java

Reply:
So the query string as argument in releaseQuery is a serialised JSON now? I wouldn't do that. I would't pass JSON as serialised string between Java objects. Why not use a generic JSON object or POJO here and serialise it right before it goes into the DB.